### PR TITLE
Updated task name so that it will work with kubernetes out of the box.

### LINKED ISSUE #210: Update Partition Sample App Name
The application name should run on k8's without modification ie. replace `PartitionBatchJobTask` with `partitionedbatchjobtask`

### PR DESCRIPTION
**Pull request by cppwfs**
resolves #210

### CODE REVIEW

File: spring-cloud-task-samples/partitioned-batch-job/src/main/java/io/spring/JobConfiguration.java [hunk 1]
```diff
@@ -104,7 +104,7 @@ public PartitionHandler partitionHandler(TaskLauncher taskLauncher, JobExplorer
 		partitionHandler.setCommandLineArgsProvider(new PassThroughCommandLineArgsProvider(commandLineArgs));
 		partitionHandler.setEnvironmentVariablesProvider(new SimpleEnvironmentVariablesProvider(this.environment));
 		partitionHandler.setMaxWorkers(1);
-		partitionHandler.setApplicationName("PartitionedBatchJobTask");
```
Comment (trisberg) [inline spring-cloud-task-samples/partitioned-batch-job/src/main/java/io/spring/JobConfiguration.java]: I added a `toLowerCase()` for the id generated in the k8s deployer, so if you prefer, you can keep the name as it was

**Comment by cppwfs** [comment]
I thought the deployer would throw an exception if it was not an properly formatted.

**Comment by trisberg** [comment]
Since it's a generated id it's not necessary to throw an exception, we can try to make it fit - https://github.com/trisberg/spring-cloud-deployer-kubernetes/blob/ce7b8d8e0a64ebfe4aee373ca29accfe903e95d6/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesTaskLauncher.java#L114-L121

**Comment by cppwfs** [comment]
Gotcha.  Closing PR as complete.

**Pull request closed without merging**